modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm

```diff
@@ -139,6 +159,14 @@
 	item_type = /obj/item/defibrillator/loaded
 	cost = PAYCHECK_COMMAND
 
+/datum/armament_entry/company_import/deforest/equipment/loaded_belt_defib
+	item_type = /obj/item/defibrillator/compact/loaded
+	cost = PAYCHECK_COMMAND*2
+
+/datum/armament_entry/company_import/deforest/equipment/loaded_belt_defib2
+	item_type = /obj/item/defibrillator/compact/combat/loaded/nanotrasen
+	cost = PAYCHECK_COMMAND*8
+
```

Comment:
There is a reason only the CMO gets this. Parameds would just run around with these completely destroying the cycle of `Go rescue body -> have MDs revive it` by performing field surgery and revival wherever there is a table, since on regular injuries that is just faster.

```suggestion
/* BUBBER EDIT REMOVAL START
/datum/armament_entry/company_import/deforest/equipment/loaded_belt_defib
	item_type = /obj/item/defibrillator/compact/loaded
	cost = PAYCHECK_COMMAND*2

/datum/armament_entry/company_import/deforest/equipment/loaded_belt_defib2
	item_type = /obj/item/defibrillator/compact/combat/loaded/nanotrasen
	cost = PAYCHECK_COMMAND*8
*/ // BUBBER EDIT REMOVAL END
```